RefrigeratorSaraFarkash/RefrigeratorSaraFarkash/CItem.cs

```diff
@@ -0,0 +1,124 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace RefrigeratorSaraFarkash
+{
+    internal class CItem:IComparable<CItem>
+    {
+
+        private static int countObj = 0;
+        private int id;
+
+        public int Id
+        {
+            get { return id; }
+            //set { id = countObj; }
+
+        }
+        private string name;
+
+        public string Name
+        {
+            get { return name; }
+            set
+            {
+                if (!value.Equals(""))
+                    name = value;
+                else
+                    throw new ArithmeticException("Illegal name, name must be string");
```

Comment:
ArithmeticException is not the correct Exception for this.There is no Arithmetic operation here.
Please find the correct Exception to use

Reply:
fixed